packages/vaadin-lumo-styles/user-colors.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import './version.js';
+import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+const userColors = css`
+  :host {
+    --vaadin-user-color-0: #df0b92;
```

Comment:
See the original Lumo styles [here](https://github.com/vaadin/collaboration-engine-internal/blob/master/collaboration-engine-frontend/field-highlighter/theme/lumo/vaadin-user-colors.js).